README.md

```diff
@@ -114,6 +114,54 @@ If you need to add different options for each country you can enter them in obje
   };
 ```
 
+### Enabling Email and Phone Validation
+```
+w.AddressFinderConfig = {
+  key: "ADDRESSFINDER_DEMO_KEY",
+
+  // Options: (true/false) to toggle each widget on/off. 
+  // The address widget is enabled by default
+  addressWidgetEnabled: true,
+  emailWidgetEnabled: true,
+  phoneWidgetEnabled: true,
+
+  // visit https://addressfinder.nz/docs/javascript_widget_reference_nz/ for a full list of NZ widget options
+  // visit https://addressfinder.com/r/docs/javascript_widget_reference_au/ for a full list of AU widget options
+  widgetOptions: {
```

Comment:
It would mean adding another pipe operation [here](https://github.com/AddressFinder/addressfinder-bigcommerce/blob/master/src/bigcommerce_plugin.js#L35-L36). We have to keep `widgetOptions` for the configurations that are already out there. Is it worth it?

Reply:
I'm good with that 👍